src/librustc_middle/mir/visit.rs

```diff
@@ -413,16 +407,10 @@ macro_rules! make_mir_visitor {
 
             fn super_terminator(&mut self,
                                 terminator: &$($mutability)? Terminator<'tcx>,
-                                location: Location) {
+                                source_location: Location) {
```

Comment:
Almost everything else uses `location`, might as well keep that.

Reply:
Oh I see, you didn't want to change all the uses of it. I guess it doesn't matter.

Reply:
Unfortunately some terminators have a field `location`, so there's a clash. I wanted to rename but then stopped half-way through due to that.

Reply:
Specifically, the two `Drop` terminators have `location: Place<'tcx>`. It's rather confusing.^^

Reply:
I'll rename those fields to `place`.